oauthenticator/oauth2.py

```diff
@@ -381,6 +382,15 @@ def _client_secret_default(self):
                 return client_secret
         return os.getenv("OAUTH_CLIENT_SECRET", "")
 
+    access_token_expiration_env = "OAUTH_ACCESS_TOKEN_EXPIRATION"
+    access_token_expiration = Unicode(
+        config=True,
+        help="""Default expiration, in seconds, of the access token."""
+    )
+
+    def _access_token_expiration_default(self):
+        return os.getenv(self.access_token_expiration_env, "3600")
+
```

Comment:
May be replaced by [Authenticator.auth_refresh_age](https://jupyterhub.readthedocs.io/en/stable/api/auth.html#jupyterhub.auth.Authenticator.auth_refresh_age) ?